src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -183,192 +188,235 @@
             </InfoBar>
         </StackPanel>
 
-        <!--
-            https://github.com/microsoft/microsoft-ui-xaml/issues/7690
-            AllowDrop="{x:Bind ViewModel.Filtered, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
-            CanDragItems="{x:Bind ViewModel.Filtered, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
-            CanReorderItems="{x:Bind ViewModel.Filtered, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
-        -->
```

Comment:
I think it makes sense to still leave the comment in, in case someone thinks about doing this feature again. We've hit it recently on MWB, for example 😅 . Had to disable drag and drop when running as admin